### ISSUE TITLE
Error about BatchNorm2dSync for 1d input in eval model, but I can use this BatchNorm2dSync for 1d input in train model

### ISSUE THREAD

**Issue by XiwuChen** [tamakoji/pytorch-syncbn#9]
Error about BatchNorm2dSync for 1d input in the  eval() model, but I can use this BatchNorm2dSync for 1d input in the train() model. Can you figure out this problem? 

Thanks,
Xiwu
